kind/config/topology.yaml

```diff
@@ -0,0 +1,23 @@
+configuration:
+  cluster_configuration:
+    kube_config_path: <KUBECONFIG_PATH>
+    controller:
+      name: controller
+      context_name: kind-controller
+    workers:
+      - name: worker-1
+        context_name: kind-worker-1
+      - name: worker-2
+        context_name: kind-worker-2      
+  kubeslice_configuration:
+    project_name: demo
+  helm_chart_configuration:
+    repo_alias: kubeslice
```

Comment:
It is using old charts repo https://kubeslice.github.io/charts/ it should point to https://kubeslice.github.io/kubeslice/

Reply:
Ok, fixed